frontend/src/components/user/ProfileCard.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import Avatar from '../Avatar'
+import { IUser } from '~/data/interfaces'
+import { classNames } from '~/utils/classNames'
+
+type Props = {
+  user: IUser
+}
+
+const ProfileCard: React.FC<Props> = (props): JSX.Element => {
+  const { user } = props
+
+  return (
+    <section className="w-1/2 min-h-[20vh]">
+      <div className="max-w-sm bg-white shadow-sm border rounded-lg py-6">
+        <div className="flex flex-col items-center pb-3">
+            <div className="inline-flex rounded-full shadow-lg">
+              <Avatar 
+                url={`${user?.avatar_url}`}
```

Comment:
maybe put a default as well since there might be a scenario where the user does not have an avatar